### ISSUE TITLE
After export the '+' is missing at the beginning of phone number in CSV

### ISSUE THREAD

**Issue by jekutzsche** [iris-connect/iris-client#443]
Looh at Marco Baumgartner with `+` in FE and without this in CSV Export.
![grafik](https://user-images.githubusercontent.com/26020101/139832911-818131de-4f6a-4439-aaa2-0857a3e9d644.png)

**Comment by jekutzsche** [comment]
@mad-nuts @bkastl 
A phone number prepended with a single quote, or also enclosed in single quotes, are imported into Sormas with these characters. If the phone number is enclosed with `"`, only the number will end up in Sormas. I think we should enclose the numbers with `"`.

**Comment by mad-nuts** [comment]
After some tests I figured out, that quoting the phone number helps but isn't sufficient to properly mitigate Formula injections in Excel. If a user imports the .csv file and double clicks into the cell with the international phone number (+123) Excel transforms the string to a Formula.

International phone numbers are processed as follows to mitigate the risk of Formula injections:
- prepend a space to the value (formerly a single qoute was prepended, to make it super safe but a space should be fine, too)
- quote the value

In addition to that I have added a sanitation that checks the value for delimiters "," or ";" and removes any Formula character that is preceded by a delimiter.
